content/blog/computer-ergonomics/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Computer ergonomics
+date: "2019-06-03T12:12:03.284Z"
+description: Don't slouch.
+---
+
+> The definition of computer ergonomics is the practice of fitting the set up of the computer and the workspace to fit the user and the user's work needs in order to minimize physical stress on the computer user (Your dictionary).
+
+Developers sit in front of a computer for hours on end. As you may already know, sitting is not the most natural position for humans to be in for a prolonged period of time. Small repetitive strain on the body from bad posture can cause people to experience muscle strain and headaches.
+
+![Back pain](./back-pain.jpg)
+
+Being aware and mindful of the body position you are in during work can help reduce or completely mitigate bad posture habits. On the contrary, just getting a shiny new expensive standing desk will not solve all your problems.
```

Comment:
_Being aware and mindful_

'aware' and 'mindful' pretty much describe the same thing. i would keep just 'aware'